elastic/kibana
# [Uptime] Index alert reason in Uptime alert documents

## Summary

Fixes #106729

This PR updates the Uptime RAC documents to include reason using the appropriate technical field. 

1. [Monitor status alerts](https://github.com/elastic/kibana/compare/master...dominiqueclarke:feature/106729-uptime-index-alert-reason-field?expand=1#diff-66e5cc62ccd453dfb324f5fa26278cf3755c91caae4150660eef74e0ef52fb9eR164)
2. [TLS Alerts](https://github.com/elastic/kibana/compare/master...dominiqueclarke:feature/106729-uptime-index-alert-reason-field?expand=1#diff-4a3072dda589206cc304a2071ebc5b2ea95f05c86cdeea13cb3c0e5f3acf6898R175)
3. [Duration anomaly alerts](https://github.com/elastic/kibana/compare/master...dominiqueclarke:feature/106729-uptime-index-alert-reason-field?expand=1#diff-f8a703e23f86f4ef9a2e171c3a254a588bc0500845abcb4557e39549efd7bf5cR176)

### Testing

This PR does not introduce any new functionality, but should be smoke tested for regressions.

1. Go to the Uptime page and create an alert rule for TLS or monitor status, ensuring that the alert will be triggered either by a down monitor or an expiring or aging certificate
2. Navigate to `observability/alerts`. Observe the Uptime alerts with the appropriate reason in the reason column in the alert table.
